src/blob_storage.h

```diff
@@ -1,7 +1,8 @@
 #pragma once
-
+#ifndef __STDC_FORMAT_MACROS
```

Comment:
seems it should in `blob_gc_picker` instead of here

Reply:
blob_storage.cc uses PRIu64 so blob_storage.h includes <inttypes.h>. blob_gc_picker.cc uses it too and it include blob_storage.h so I think it's not necessary to include in blob_gc_picker.h, is it right? or I need include this file in picker also?